pkg/pipelines/init.go

```diff
@@ -213,6 +214,18 @@ func createCICDResources(fs afero.Fs, repo scm.Repository, cicdEnv *config.Envir
 	return outputs, nil
 }
 
+// Trigger bindings for all the repository types will be created during bootstrap
+func createTriggerBindings(outputs res.Resources, ns string) {
```

Comment:
This doesn't look quite right.

The fact that you're _always_ dropping the second return value from CreatePRBinding should be a smell that something isn't correct.

I suspect you want to use that with the binding path to ensure that the files get written to an appropriate location.

Reply:
CreatePRBinding returns the binding and its name. We just need the name of the binding in certain cases and hence the other value is dropped. I guess this can be solved by returning only the binding and extracting the name from it